src/tests/nunit.runner.tests.Droid/MainActivity.cs

```diff
@@ -43,7 +45,10 @@ protected override void OnCreate(Bundle savedInstanceState)
             //nunit.AddTestAssembly(typeof(MyTests).Assembly);
 
             // Do you want to automatically run tests when the app starts?
-            nunit.AutoRun = true;
+            nunit.Options = new TestOptions { AutoRun = true };
```

Comment:
Minor point, but rather than have a commented out available options below, I would prefer something like,

```C#
nunit.Options = new TestOptions
{
  // Automatically run the tests when the app starts
  AutoRun = true,  

  // Report test results back to the server
  // TcpWriterParamaters = new TcpWriterInfo("10.0.2.2", 13000),

  // Save the test results to local storage
  CreateXmlResultsFile = false  
};
```